apps/api/src/app/access/create-access.dto.ts

```diff
@@ -6,6 +8,9 @@ export class CreateAccessDto {
   @IsString()
   alias?: string;
 
+  @IsOptional()
```

Comment:
`@IsObject()` ❓

Reply:
Should `@isOptional()` be removed?. In this case form is not accepted if there is no filter configured. I add `@IsObject()` decorator before `@isOptional()`